### PR TITLE
Added plugin for desk

### PR DESCRIPTION
**Pull request by mblarsen**
Added completion of available desks for [desk](https://github.com/jamesob/desk).

### CODE REVIEW

File: plugins/desk/desk.plugin.zsh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    done
+
+    _arguments \
+      '1:command:->cmds' \
+      '2:desk:->desk'
+
+    case $state in
+        (cmds)
+            _values "desk commands" \
+                'go[Activate a desk]' \
```
Comment (jamesob) [inline plugins/desk/desk.plugin.zsh]: `.` is also synonymous with `go`

File: plugins/desk/desk.plugin.zsh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                'init[Initialize desk configuration]' \
+                'list[List all desks along with description]'
```
Comment (jamesob) [inline plugins/desk/desk.plugin.zsh]: `ls` as well

File: plugins/desk/desk.plugin.zsh (hunk 1 above)
Comment (mblarsen) [inline plugins/desk/desk.plugin.zsh]: Added

File: plugins/desk/desk.plugin.zsh (hunk 2 above)
Comment (mblarsen) [inline plugins/desk/desk.plugin.zsh]: Added

**Comment by jamesob** [comment]
Nice! LGTM

**Comment by mcornella** [comment]
I know this is old but now the project includes a [guide to set it up within OMZ](https://github.com/jamesob/desk#using-oh-my-zsh) so I don't think we need a plugin.

**Pull request closed without merging**